File: Day 6: Let's Review/Solution.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
  /* Enter your code here. Read input from STDIN. Print output to STDOUT */
  int T;
  vector<string> Slist;
  cin >> T;
  for (int i = 0; i < T; i++)
  {
    /* code */
    string S;
    cin >> S;
    Slist.push_back(S);
  }

  for (int i = 0; i < Slist.size(); i++)
  {
    string first;
    string second;
    for (int j = 0; j < Slist[i].length(); j++)
    {
      if (j % 2 == 0)
      {
        first = first + Slist[i][j];
      }
      else
      {
        second = second + Slist[i][j];
      }
    }
    cout << first << " ";
    cout << second << endl;
  }

  return 0;
}

File: Day 6: Let's Review/Solution.py
# Enter your code here. Read input from STDIN. Print output to STDOUT

#!/bin/python3

import math
import os
import random
import re
import sys



if __name__ == '__main__':
    T = int(input())
    Slist = []
    for i in range(0,T):
      Slist.append(input())

    for i in range(0, T):
      first = ""
      second = ""
      for j in range(0, len(Slist[i])):
        if j%2 == 0:
          first += Slist[i][j]
        else:
          second += Slist[i][j]
      print("{} {}".format(first,second))
